demos/online-boutique/OLJCESPC7Z

```diff
@@ -0,0 +1,229 @@
+
```

Comment:
Is this file used for test data, or was it created accidentally? If it's the latter perhaps we could move it to a `testdata` folder.

Reply:
Create by mistake. Got catch.